kie-wb-webapp-common/src/main/webapp/login.jsp

```diff
@@ -6,16 +6,18 @@
 
 <%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
 <%@ taglib uri="http://jakarta.apache.org/taglibs/i18n-1.0" prefix="i18n" %>
+
 <%
-  Locale locale= null;
-  try{
-    locale = LocaleUtils.toLocale( request.getParameter("locale") );
-  } catch(Exception e){
-    locale= request.getLocale();
+  Locale locale = null;
+  try {
+    locale = LocaleUtils.toLocale(request.getParameter("locale"));
+  } catch (Exception e) {
+    locale = request.getLocale();
   }
-  SecureHeadersFilter.applyHeaders( request, response );
+  SecureHeadersFilter.applyHeaders(request,
+                                   response);
 %>
-<i18n:bundle id="bundle" baseName="org.kie.workbench.drools.client.resources.i18n.LoginConstants"
+<i18n:bundle id="bundle" baseName="${login.bundle.name}"
```

Comment:
Correct, this is done via maven overlay with filtering enabled: https://github.com/cristianonicolai/kie-wb-distributions/blob/JBPM-6148-7.2.x/kie-drools-wb-parent/kie-drools-wb-webapp/pom.xml#L1905